### DISCUSSION TITLE
OAUTH 2.0, OpenID connect, SSO for Google, Apple, etc

### DISCUSSION THREAD

**Discussion by edardev** [medusajs/medusa#2511] [RFC]
Allow 3rd parties authenticator services like Auth0, AWS Conigto, Custom auth server, etc....

**Comment by srindom** [comment]
@adrien2p recently published this: https://github.com/adrien2p/medusa-plugins/tree/main/packages/medusa-plugin-auth

Maybe that can serve as inspiration for a custom auth handler

**Reply by adrien2p** [reply]
@edardev do not hesitate to open a pr to add other auth platforms or providers. I am planning to add a bunch of them and would be more than happy to get some hand to contribute to it😍

**Comment by lithiumlab** [comment]
SAML seems another good candidate for an MedusaJS Auth Handler. Please consider.
